README.md

```diff
@@ -1,22 +1,78 @@
+  - [Styles API](#styles-api)
+- [Troubleshooting](#troubleshooting)
+- [Tests](#tests)
 
+## Usage
 
-## Continuous Integrations 
+This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
 
-### Travis CI 
-[![Build Status](https://travis-ci.org/vtex-apps/category-menu.svg?branch=master)](https://travis-ci.org/vtex-apps/category-menu)
+To use this app you need to add it in your dependencies in the `manifest.json` file.
 
-## Usage
+```json
+  dependencies: {
+    "vtex.category-menu": "2.x"
+  }
+```
+
+Then, add `category-menu` block into your app theme, like we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 
+
+### Blocks API
+:construction: :construction: :construction:
+
+This app has an interface that describes what rules must be implemented by a block when you want to use the category-menu block.
+
+```json
+  "category-menu": {
+    "component": "index"
+  }
+}
+```
+
+#### Configuration 
+Through the Storefront you can change the behavior and interface of category-menu. But, you can also make adjusts in your theme app, like Dreamstore does.
+
+| Prop name          | Type       | Description                                                                 |
+| ------------------ | ---------- | --------------------------------------------------------------------------- |
+| `showPromotionCategory`                      | `Boolean`   | Shows the promotion category in menu               |
+| `showGiftCategory`          | `Boolean`  | Shows the gift category in menu                               |
+| `showAllDepartments`              | `Boolean`  | Shows all departments category in menu                              |
+| `menuDisposition`                   | `Enum`  | Indicates the disposition of the menu on the screen (left, center, right)                                     |
+| `showSubcategories`        | `Boolean`   | Decides if the subcategories will be displayed
+| `departments` | `Array(items)`   | List of departments `items` to be displayed on the menu  |
+
+Items:
 
-TODO
+| Prop name          | Type       | Description                                                                 |
+| ------------------ | ---------- | --------------------------------------------------------------------------- |
+| `id`                      | `Number`   | The department Id to be displayed on the menu               |
+
+### Styles API
+:construction: :construction: :construction:
+
+## Troubleshooting
+You can check if others are passing through similar issues [here](https://github.com/vtex-apps/category-menu/issues). Also feel free to [open issues](https://github.com/vtex-apps/category-menu/issues/new) or contribute with pull requests.
+
+## Tests
+:construction: :construction: :construction:
```

Comment:
Use the same `Tests` as [Minicart](https://github.com/vtex-apps/minicart#tests).